demisto_sdk/commands/lint/lint_manager.py

```diff
@@ -374,8 +374,11 @@ def execute_all_packages(self,
                                                    test_xml=test_xml,
                                                    no_coverage=no_coverage))
 
-                for future in concurrent.futures.as_completed(results):
+                logger.info('Waiting for futures to complete')
+                for i, future in enumerate(concurrent.futures.as_completed(results)):
+                    logger.info(f'Waiting for future index {i}')
```

Comment:
does `debug` suit better? to avoid spamming info, at least for some of the calls here.